src/Tribe/Tickets.php

```diff
@@ -1658,6 +1658,11 @@ public static function get_attendees_from_modules( $attendees, $post_id = 0 ) {
 					continue;
 				}
 
+				// Skip if the provider is Tickets Commerce but the module is disabled.
+				if ( $provider instanceof TEC\Tickets\Commerce\Module && ! tec_tickets_commerce_is_enabled() ) {
+					continue;
+				}
```

Comment:
Look into these [lines](https://github.com/the-events-calendar/event-tickets/blob/93792c7774e9c55ad82206a79d7a4593293a1f14/src/Tribe/Tickets.php#L1655-L1660) above. Here it should return false for `tribe_tickets_get_ticket_provider` for disabled TicketsComemrce provider.